release notes/v0.53.0.md

```diff
@@ -0,0 +1,295 @@
+
+ECMAScript Modules(ESM) is the last such feature that Babel was used for. It also happens to be likely the one *most* people used, due to the fact that it is the standard way to reuse code and import libraries.
+
+While the work on this feature started over 2 years ago, it both depended on other features that weren't there yet, but also interacts with more or less every other feature that is part of the ECMAScript standard.
+
+Along the way there were many internal refactors as well as additional tests to make certain we can be as backwards compatible as possible. But there also ended up being things that just weren't going to be compatible, like the listed breaking changes.
+
+</details>
+
+After ESM now being natively supported, compatibility-mode `base` vs `extended` has only 1 feature difference - aliasing `global` to `globalThis` to make it a bit more compatible with (old) Node.js. There is ongoing [discussion](https://github.com/grafana/k6/issues/3864) if that as well should be removed.
+
+For the purposes of having less intrusive changes and shipping this earlier a few things have not been implemented in k6. That includes top-level-await and dynamic import support. Both of them are likely to land in the next version.
+
+### `import.meta.resolve()` get URL from a relative path the same way `import` or `require` does [#3873](https://github.com/grafana/k6/pull/3873)
+
+As part of the move to ESM a lot of cases where k6 currently do not resolve the same relative path to the same file were found. Some of those were fixed - as those in `require`, but others haven't.
+
+It also became apparent some users do use the relativity of `require`, but also `open`. As we move to make this consistent among uses, we decided to let users have a better transition path forward.
+
+Using `import.meta.resolve` will give you just a new URL that can be used in all functions and it will give you the same result.
+
+`import.meta.resolve` uses the same algorithm and relativity as ESM import syntax.
+
+### Blob support in the experimental websockets module [grafana/xk6-websockets#74](https://github.com/grafana/xk6-websockets/pull/74)
+
+In order to support the default `WebSocket.binaryType` type as per spec (`"blob"`),  we have added support for the [`Blob` interface](https://developer.mozilla.org/en-US/docs/Web/API/Blob) as part of the features included in the `xk6-websockets` module. 
+
+So, from now on it can be used with `import { Blob } from "k6/experimental/websockets";`. In the future, apart from graduating this module to stable, we might also want to expose the `Blob` interface globally (no imports will be required). But for now, please remind that its support is still experimental, as the entire module is.
```

Comment:
```suggestion
So, from now on it can be used with `import { Blob } from "k6/experimental/websockets";`. In the future, apart from graduating this module to stable, we might also want to expose the `Blob` interface globally (no imports will be required). But for now, please remind that its support is still experimental, as the entire module is. Refer to the [docs](https://grafana.com/docs/k6/next/javascript-api/k6-experimental/websockets/blob/) for more details.
```

That should be the correct URL once https://github.com/grafana/k6-docs/pull/1690 is merged.